src/EpisodeIntegrationService/ReceiveData/ReceiveData.cs

```diff
@@ -272,7 +272,7 @@ private static InitialEpisodeDto MapEpisodeToEpisodeDto(BssEpisode episode)
             EarlyRecallDate = Utils.ParseNullableDate(episode.early_recall_date),
             CallRecallStatusAuthorisedBy = episode.call_recall_status_authorised_by,
             EndCode = episode.end_code,
-            EndCodeLastUpdated = Utils.ParseNullableDateTime(episode.end_code_last_updated, "yyyy-MM-dd HH:mm:ssz"),
+            EndCodeLastUpdated = Utils.ParseNullableDateTime(episode.end_code_last_updated, new[] {"yyyy-MM-dd H:mm:ssz", "yyyy-MM-dd H:mm:ss"}),
```

Comment:
Suggest `HH` rather than `H` as the leading zero helps with sorting:

```suggestion
            EndCodeLastUpdated = Utils.ParseNullableDateTime(episode.end_code_last_updated, new[] {"yyyy-MM-dd HH:mm:ssz", "yyyy-MM-dd HH:mm:ss"}),
```